org.pitivi.Pitivi.json

```diff
@@ -0,0 +1,497 @@
+                    "branch": "0.8",
+                    "commit": "9d2ac65a2f88263f0eec8b777ef3bdc023b93c25"
+                }
+            ]
+        },
+        {
+            "name": "gsound",
+            "ensure-writable": ["easy-install.pth"],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://git.gnome.org/browse/gsound",
+                    "branch": "1.0.2",
+                    "commit": "a86946b494aa32782b3f6334344d07b5c11d5219"
+                }
+            ]
+        },
+        {
+            "name": "x264",
+            "config-opts": ["--enable-shared", "--enable-static", "--enable-pic", "--disable-lavf"],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://download.videolan.org/pub/x264/snapshots/x264-snapshot-20140212-2245-stable.tar.bz2",
+                    "sha256": "5d98e9e4faf6dd55e7193ed379aff477b8acbda6777758956ef7e5f05067be18"
+                }
+            ]
+        },
+        {
+            "name": "libopus",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://downloads.xiph.org/releases/opus/opus-1.1.2.tar.gz",
+                    "sha256": "0e290078e31211baa7b5886bcc8ab6bc048b9fc83882532da4a1a45e58e907fd"
+                }
+            ]
+        },
+        {
+            "name": "frei0r-plugin",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://files.dyne.org/frei0r/releases/frei0r-plugins-1.4.tar.gz",
+                    "sha256": "8470fcabde9f341b729be3be16385ffc8383d6f3328213907a43851b6e83be57"
+                }
+            ]
+        },
+        {
+            "name": "libsoundtouch",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://www.surina.net/soundtouch/soundtouch-1.8.0.tar.gz",
+                    "sha256": "3d4161d74ca25c5a98c69dbb8ea10fd2be409ba1a3a0bf81db407c4c261f166b"
+                }
+            ]
+        },
+        {
+            "name": "lame",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://downloads.sourceforge.net/lame/lame-3.99.5.tar.gz",
+                    "sha256": "24346b4158e4af3bd9f2e194bb23eb473c75fb7377011523353196b19b9a23ff"
+                }
+            ]
+        },
+        {
+            "name": "vo-aacenc",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://downloads.sourceforge.net/project/opencore-amr/vo-aacenc/vo-aacenc-0.1.3.tar.gz",
+                    "sha256": "e51a7477a359f18df7c4f82d195dab4e14e7414cbd48cf79cc195fc446850f36"
+                }
+            ]
+        },
+        {
+            "name": "x265",
+            "cmake": true,
```

Comment:
Please use `"buildsystem": "cmake"`, `cmake` as key is marked as deprecated.

Reply:
Done.